### PR TITLE
chore: add cicd (devplat) team to codeowners

### PR DESCRIPTION
**Pull request by Shabirmean**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-cloudbuild/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [X] Code coverage does not decrease (if any source code was changed)
- [X] Appropriate docs were updated (if necessary)

Fixes #NA 🦕

### CODE REVIEW

File: .repo-metadata.json [hunk 1]
```diff
@@ -10,5 +10,6 @@
   "repo": "googleapis/python-cloudbuild",
   "distribution_name": "google-cloud-build",
   "api_id": "cloudbuild.googleapis.com",
+  "codeowner_team": "@googleapis/cicd",
```
Comment (busunkim96) [inline .repo-metadata.json:13]: @Shabirmean The python repos don't currently template the CODEOWNERS file, so you'll also need to add the team to https://github.com/googleapis/python-cloudbuild/blob/master/.github/CODEOWNERS

**Review by busunkim96 — APPROVED** [review]

File: .repo-metadata.json (hunk 1 above)
Comment (busunkim96) [reply .repo-metadata.json:13]: Ah ignore me, I see the team is already in the list. :) You might want to also add `cicd` to `/samples`

Comment (Shabirmean) [reply .repo-metadata.json:13]: @busunkim96 - they seem to have been [updated already](https://github.com/googleapis/python-cloudbuild/blob/master/.github/CODEOWNERS#L8); so I left them as is

Comment (Shabirmean) [reply .repo-metadata.json:13]: @busunkim96 Done. Thank you for reviewing 🙌🏼

**Pull request merged**